### PR TITLE
Add Docplex tutorials

### PR DESCRIPTION
**Pull request by MaldoAlberto**
## Description

-Three examples of how encoding an optimization problem in openQAOA using a docplex model.

 - Tutorial 10  Basic example
 - Tutorial 11  Portfolio Optimization
 - Tutorial 12 Bin Packing problem
-using the class FromDocplex2IsingModel
- Tutorial about QAOA using OpenQAOA

- **Fixes # (issue)**

## Checklist

[//]: <> (- [ ] My code follows the style guidelines of this project)

- [ ] I have performed a self-review of my code.
- [ ] I have commented my code and used numpy-style docstrings
- [ ] I have made corresponding updates to the documentation.
- [ ] My changes generate no new warnings
- [ ] I have added/updated tests to make sure bugfix/feature works.
- [ ] New and existing unit tests pass locally with my changes.

[//]: <> (- [ ] Any dependent changes have been merged and published in downstream modules)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Name the new unit-tests that you have added along with this change.

### CODE REVIEW

File: examples/Tutorail 1 Quantum Approximate Optimization Algorithm.ipynb [hunk 1]
```diff
@@ -0,0 +1,643 @@
+{
```
Comment (Q-lds) [inline examples/Tutorail 1 Quantum Approximate Optimization Algorithm.ipynb:1]: The title should be changed to `examples/01_tutorial_quantum_approximate_optimization_algorithm.ipynb`, thanks!

**Comment by Q-lds** [comment]
@MaldoAlberto , apologies for the delay!

Could you please add the notebooks to the test file `openqaoa/tests/test_notebooks.py`?

For example, 
```
# @pytest.mark.notebook
def test_10_docplex_example():
    notebook_test_function("./examples/10_docplex_example.ipynb")
```

Thanks!

**Comment by MaldoAlberto** [comment]
@Q-lds  I added the changes that you indicated me :smile:

**Comment by Q-lds** [comment]
@vishal-ph, here we have two open questions:
- where do we put the dependencies of the community notebooks? _[maybe an extra pip install flag so that they are installed only when truly needed?]_

- shall we move the notebooks to `examples/community_tutorials/`? _[I'd say yes, but we can move them in a future PR. Otherwise, we need to fix the names right now and ask Alberto to move them. I'd say we can move them ourselves, as it will be easier than writing it here!]_

**Pull request merged**